app/components/contents/external_file_component.html.erb

```diff
@@ -5,8 +5,7 @@
         External File
       </span>
       <%= filename %>
```

Comment:
I think filename can be removed too

Reply:
@jcoyne I updated the spec, removed filename and added attr_reader for druid. I guess I should see how this works in stage before pulling out of draft?